src/main/java/com/testaccounting/service/CsvService.java

```diff
@@ -0,0 +1,71 @@
+package com.testaccounting.service;
+
+import com.opencsv.CSVReader;
+import com.opencsv.exceptions.CsvValidationException;
+import com.testaccounting.data.entity.Currency;
+import com.testaccounting.helper.Constant;
+import org.springframework.data.util.Pair;
+import org.springframework.stereotype.Service;
+
+import java.io.IOException;
+import java.math.BigDecimal;
+import java.time.LocalDate;
+import java.time.Month;
+
+@Service
+public class CsvService {
+    private final EmployeeService employeeService;
+    private final RatesService ratesService;
+
+    public CsvService(EmployeeService employeeService, RatesService ratesService) {
+        this.employeeService = employeeService;
+        this.ratesService = ratesService;
+    }
+
+    public void parseCsvAndSaveEmployeesAndSalaries(String fileName, CSVReader csvReader)throws IOException, CsvValidationException {
+        Pair<Month, Integer> my = this.parseEmployeeSalaryFileNameAndGetYearAndMonth(fileName);
+        Month m = my.getFirst();
+        Integer y = my.getSecond();
+        int lineNum = 0;
+        String[] line;
+        while ((line = csvReader.readNext()) != null) {
```

Comment:
List<String> xmlLines = new BufferedReader(new FileReader(csvFile))
    .lines()
    .skip(1) //Skips the first n lines, in this case 1      
    .map(s -> {
        //csv line parsing and xml logic here
        //...
        return whatever;
    })
    .collect(Collectors.toList());